azure-pipelines.yml

```diff
@@ -1,20 +1,28 @@
-name: HMCTS CFT AKS Azure Infrastructure Deployment Pipeline
+name: "${{ parameters.action }} - ${{ parameters.env }}.${{ parameters.cluster }}.${{ parameters.location }}"
 
 trigger: none
 
 pool:
   vmImage: 'ubuntu-latest'
 
+resources:
+  repositories:
+    - repository: cnp-azuredevops-libraries
+      type: github
+      ref: refs/heads/DTSPO-9695-make-tf-varfile-configurable
```

Comment:
can we ensure this branch has been tested against other pipelines

Reply:
Pipeline for `azure-private-dns` has been tested using the new common pipeline template changes https://dev.azure.com/hmcts/CNP/_build/results?buildId=279519&view=results

I'll test a few more pipelines just to be on the safe side